controller/filter.go

```diff
@@ -41,16 +42,27 @@ const (
 // List runs the list action.
 func (c *FilterController) List(ctx *app.ListFilterContext) error {
 	var arr []*app.Filters
-	arr = append(arr, &app.Filters{
-		Attributes: &app.FilterAttributes{
-			Title:       "Assignee",
-			Description: "Filter by assignee",
-			Type:        "users",
-			Query:       fmt.Sprintf("filter[%s]={id}", filterKeyAssignee),
-			Key:         filterKeyAssignee,
+	arr = append(arr,
+		&app.Filters{
+			Attributes: &app.FilterAttributes{
+				Title:       "Assignee",
+				Description: "Filter by assignee",
+				Type:        "users",
+				Query:       fmt.Sprintf("filter[%s]={id}", filterKeyAssignee),
+				Key:         filterKeyAssignee,
+			},
+			Type: "filters",
+		},
+		&app.Filters{
+			Attributes: &app.FilterAttributes{
+				Title:       "Creator",
+				Description: "Filter by creator",
+				Type:        "users",
+				Query:       fmt.Sprintf("filter[%s]={id}", filterKeyAssignee),
```

Comment:
I think you meant `filterKeyCreator` here

Reply:
Corrected 0abab45d4fa5a16a1fc7b16873bec2158d6477f3